components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp

```diff
@@ -105,14 +104,14 @@
 					<div class="control-group">
 					   <% if (loginFailed) { %>
                           <div class="alert alert-error">
-                              <%=errorMessage%>
+                              <%=Encode.forHtmlContent(errorMessage)%>
```

Comment:
&,<,> are escaped using forHtmlContent().
Do we need to escape ' and " inside dev.